src/guiapi/include/term.h

```diff
@@ -37,17 +45,26 @@
 #define TERM_PRINTF_MAX 0xff
 
 typedef struct _term_t {
+    // 16 bit
```

Comment:
Don't reorder members unless you know what you are doing!
flg 2B
size 2B
cols 1B
rows 1B -> 2B alignment space for the 4B buff ptr